src/RoutableInterface.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Littledev\IPTools;
+
+interface RoutableInterface
```

Comment:
If you add the address function maybe is `addressable` a better term, since we don't actually do routing and such.

Reply:
Good point changed it. I was thinking a network and a address are both routable but we don't do routing.